src/screens/CreatePost/index.js

```diff
@@ -789,6 +791,7 @@ const CreatePost = () => {
                 setIsPollMultipleChoice(ismultiplechoice)
               }
               expiredobject={postExpired[expiredSelect].expiredobject}
+              isAnonimity={typeUser}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, please ensure that the `isAnonimity` prop is being used in the component.
```diff
-            isAnonimity={typeUser}
+            isAnonymous={typeUser}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->